ex_feels/assets/js/timefeels.js

```diff
@@ -0,0 +1,445 @@
+    .call(xAxis);
+
+  axes.append('g')
+    .attr('class', 'y axis')
+    .call(yAxis)
+    .append('text')
+      .attr('transform', 'rotate(-90)')
+      .attr('y', 6)
+      .attr('dy', '.71em')
+      .style('text-anchor', 'end')
+      .text('Time (s)');
+}
+
+function drawPaths (svg, data, x, y) {
+  var upperOuterArea = d3.svg.area()
+    .interpolate('basis')
+    .x (function (d) { return x(d.date) || 1; })
+    .y0(function (d) { return y(d.pct95); })
+    .y1(function (d) { return y(d.pct75); });
+
+  var upperInnerArea = d3.svg.area()
+    .interpolate('basis')
+    .x (function (d) { return x(d.date) || 1; })
+    .y0(function (d) { return y(d.pct75); })
+    .y1(function (d) { return y(d.pct50); });
+
+  var medianLine = d3.svg.line()
+    .interpolate('basis')
+    .x(function (d) { return x(d.date); })
+    .y(function (d) { return y(d.pct50); });
+
+  var lowerInnerArea = d3.svg.area()
+    .interpolate('basis')
+    .x (function (d) { return x(d.date) || 1; })
+    .y0(function (d) { return y(d.pct50); })
+    .y1(function (d) { return y(d.pct25); });
+
+  var lowerOuterArea = d3.svg.area()
+    .interpolate('basis')
+    .x (function (d) { return x(d.date) || 1; })
+    .y0(function (d) { return y(d.pct25); })
+    .y1(function (d) { return y(d.pct05); });
+
+  svg.datum(data);
+
+  svg.append('path')
+    .attr('class', 'area upper outer')
+    .attr('d', upperOuterArea)
+    .attr('clip-path', 'url(#rect-clip)');
+
+  svg.append('path')
+    .attr('class', 'area lower outer')
+    .attr('d', lowerOuterArea)
+    .attr('clip-path', 'url(#rect-clip)');
+
+  svg.append('path')
+    .attr('class', 'area upper inner')
+    .attr('d', upperInnerArea)
+    .attr('clip-path', 'url(#rect-clip)');
+
+  svg.append('path')
+    .attr('class', 'area lower inner')
+    .attr('d', lowerInnerArea)
+    .attr('clip-path', 'url(#rect-clip)');
+
+  svg.append('path')
+    .attr('class', 'median-line')
+    .attr('d', medianLine)
+    .attr('clip-path', 'url(#rect-clip)');
+}
+
+function startTransitions (chartWidth, rectClip) {
+  rectClip.transition().attr('width', chartWidth);
+}
+
+function makeChart (data) {
+  var svgWidth  = 960,
+      svgHeight = 500,
+      margin = { top: 20, right: 20, bottom: 40, left: 40 },
+      chartWidth  = svgWidth  - margin.left - margin.right,
+      chartHeight = svgHeight - margin.top  - margin.bottom;
+
+  var x = d3.time.scale().range([0, chartWidth])
+            .domain(d3.extent(data, function (d) { return d.date; })),
+      y = d3.scale.linear().range([chartHeight, 0])
+            .domain([0, d3.max(data, function (d) { return d.pct95; })]);
+
+  var xAxis = d3.svg.axis().scale(x).orient('bottom')
+                .innerTickSize(-chartHeight).outerTickSize(0).tickPadding(10),
+      yAxis = d3.svg.axis().scale(y).orient('left')
+                .innerTickSize(-chartWidth).outerTickSize(0).tickPadding(10);
+
+  var svg = d3.select('body').append('svg')
```

Comment:
Lets add the chart to a div via an id, `#timefeels`. The best place for the chart is probably below the header and before the tweets start.